jekyll_files/docs/admin-documentation/index.md

```diff
@@ -12,3 +12,7 @@ has_children: true
 The following guides detail how to set up the Laserfiche/SharePoint integration. Unlike the user guides, the admin guides
 may require you to hold certain privileges, such as the ability to edit a page on a SharePoint site. Each guide contains important
 information to help you set up the integration. If you are setting up for the first time, we recommend starting at the top.
+
```

Comment:
this might be a me thing so we can have Roger review it but I would put an Oxford comma
"Do not load, configure, or serve untrusted..."